openshift/e2e-common.sh

```diff
@@ -248,6 +248,10 @@ EOF
 }
 
 function create_configmaps(){
+  # Keep this until OCP 4.12.
+  # Check https://github.com/openshift/knative-serving/pull/1037#issuecomment-1021827931 for more.
+  sed -i -e "s|policy/v1|policy/v1beta1|g" openshift/release/knative-serving-ci.yaml
```

Comment:
Looking at at update-to-head.sh script, it almost looks like [this line](https://github.com/openshift/knative-serving/blob/main/openshift/release/update-to-head.sh#L22) should be moved after [this line](https://github.com/openshift/knative-serving/blob/main/openshift/release/update-to-head.sh#L15) and then this little hack in tests would not be required. In other words: we already have the patch there so why not apply it before generating the knative-serving-ci.yaml ? I didn't try it but I think this should be the way to go.

Reply:
The patch we have is [this one](https://github.com/openshift/knative-serving/blob/main/openshift/patches/006-pdb.patch). How is that related? Why order matters in moving those lines in ` update-to-head.sh` could you elaborate?
...
```

Reply:
Yeah. Right, we need to apply the patch before calling generate-release.sh. I'm not sure why we had applying the patch later but it's probably the first time we're hitting this type of issue, as you mentioned.